server/controllers/documentController.js

```diff
@@ -1,155 +1,163 @@
 import { Document } from '../models';
-import errorMessages from '../constants/errors';
-import successMessages from '../constants/successes';
-import helpers from '../helpers/helpers';
+import errorConstants from '../constants/errorConstants';
+import successConstants from '../constants/successConstants';
+import documentHelpers from '../helpers/documentHelpers';
 
-const getPageMetadata = helpers.getPageMetadata;
-export default {
+const getPageMetadata = documentHelpers.getPageMetadata;
+const documentControllers = {
+  /**
+   * @description creates a document. accepts title, content
+   * and access. responds with a created document object if 
+   * document creation succeeds
+   * @param {object} request http request object from express
+   * @param {object} response http response object  from expressjs
+   * @returns {Promise} promise from express http response
+   */
   createDocument: (request, response) => {
     const { title, content, access } = request.body;
-    const { id, role } = response.locals.user;
-    return Document.create({ title, content, access, role, author: id })
-      .then(doc =>
-        response.status(201).json({
-          document: doc.dataValues
-        })
+    const user = response.locals.user;
+    const authorId = response.locals.user.id;
+    return Document
+      .create({ title, authorId, content, access, roleId: user.roleId })
```

Comment:
Why are you not validating user's request.body fields?

Reply:
the validation is in my models. so validating twice would lead to redundancy. if you see the whole block of code, you'd see a catch block where I catch all possible validation errors